Source/Core/DolphinQt/Config/GameConfigWidget.h

```diff
@@ -50,8 +51,10 @@ class GameConfigWidget : public QWidget
   ConfigBool* m_use_monoscopic_shadows;
 
   ConfigStringChoice* m_deterministic_dual_core;
-  ConfigSlider* m_depth_slider;
-  ConfigInteger* m_convergence_spin;
+  ConfigFloatSlider* m_depth_slider;
+  ConfigFloatSlider* m_convergence_spin;
```

Comment:
```suggestion
  ConfigFloatSlider* m_convergence_slider;
```

The "_spin" suffix is no longer accurate.
I'd say "_slider" or naming them something like `m_3d_depth` and `m_3d_convergence` is fine.